symengine/codegen.h

```diff
@@ -155,7 +155,7 @@ class CodePrinter : public BaseVisitor<CodePrinter, StrPrinter>
     void bvisit(const Constant &x)
     {
         if (eq(x, *E)) {
-            str_ = "M_E";
+            str_ = "exp(1)";
         } else if (eq(x, *pi)) {
             str_ = "M_PI";
```

Comment:
somewhat unrelated: @isuruf should we print 4*atan(1) instead of `M_PI` as well then?

Reply:
How about `acos(-1)`?

Reply:
sounds good!